### PR TITLE
remove bad white-space rule

### PR DESCRIPTION
**Pull request by eluberoff**
breaks sums in FF. discussed here: https://github.com/mathquill/mathquill/commit/32a48e4a4f2cd3ecaedcf36faac5c84fe590acc9#commitcomment-11483930

### CODE REVIEW

**Comment by laughinghan** [comment]
Woah, removing `white-space` entirely? Doesn't that break pressing space more than once in a row?

**Comment by eluberoff** [comment]
good call. I'll improve this.

**Comment by eluberoff** [comment]
@laughinghan, what would you think of just using &nbsp; for spaces in rendering as a fix there?

`LatexCmds[' '] = LatexCmds.space = bind(VanillaSymbol, '\\ ', '&nbsp;');`

that seems like it's more semantic html. I'll push that to this branch

File: src/css/editable.less [hunk 1]
```diff
@@ -5,7 +5,6 @@
     width: 100%;
     padding: 2px;
     .box-sizing(border-box); // supported by IE8+: http://caniuse.com/css3-boxsizing
-    white-space: pre;
```
Comment (laughinghan) [inline src/css/editable.less]: `white-space` defaults to `normal` which causes things to wrap:
image
but if we change this line to `white-space: nowrap` instead, with the `&nbsp;` change, I can't think of any further problems.

Comment (eluberoff) [inline src/css/editable.less]: does this definitely belong just in mq-editable-field? what happens for statically rendered math?

Comment (laughinghan) [inline src/css/editable.less]: great question, hadn't thought of that

**Comment by laughinghan** [comment]
This looks good but I'd also like to add a visual test case so we can check that this won't happen in Firefox in the future (first attempt didn't work, come back to this)

File: src/css/editable.less (hunk 1 above)
Comment (laughinghan) [inline src/css/editable.less]: thanks for pointing this is out---it's exactly why my first attempt a test case for this PR didn't work

**Pull request merged**